rsmtool/rsmpredict.py

```diff
@@ -68,11 +70,14 @@ def fast_predict(
         features on which the model was trained and the values should
         be the *raw* feature values.
     modeler : rsmtool.modeler.Modeler object
-        The RSMTool modeler object using which the predictions
-        are to be generated. This object should be created from the already
-        existing model file in the "output" directory of the previously run
-        RSMTool experiment.
-    df_feature_info : pandas DataFrame
+        The RSMTool modeler object using which the predictions are to be
+        generated. This object should be created from the already existing
+        model file in the "output" directory of the previously run RSMTool
+        experiment.
+    df_feature_info : pandas DataFrame, optional
```

Comment:
This is such an elegant solution to avoid the clunkiness of storing separate feature info files alongside the model. Great idea!